modules/Users/controller.php

```diff
@@ -43,6 +43,7 @@
 }
 
 require_once("include/OutboundEmail/OutboundEmail.php");
+require_once("modules/UserPreferences/UserPreference.php");
```

Comment:
```suggestion
require_once __DIR__ . '/../../include/OutboundEmail/OutboundEmail.php';
require_once __DIR__ . '/../../modules/UserPreferences/UserPreference.php';
```

These should be changed to single quotes and use ```__DIR__``` for safer file inclusion.